src/components/daydream-controls.js

```diff
@@ -83,7 +83,9 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var isPresent = this.hands.reduce(function (present, hand) {
+      return present && this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: hand});
+    }.bind(this));
```

Comment:
@machenmusik Yes that's right.  OK, maybe the approach here is to get tracked-controls to work without the hand property.